srcpkgs/osm-gps-map/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'osm-gps-map'
+pkgname=osm-gps-map
+version=1.1.0
+revision=1
+build_style=gnu-configure
+configure_args="--enable-gtk-doc"
+hostmakedepends="gtk+3-devel libsoup-devel libtool gnome-common gtk-doc gobject-introspection pkg-config"
```

Comment:
gtk+3-devel libsoup-devel should be in makedepends=

See the manual for the difference between checkdepends, depends, makedepends and hostmakedepends